charts/document-engine/ci/01-no-storage-values.yaml

```
@@ -3,8 +3,7 @@ apiAuth:
   secretKeyBase: verySecretStringToNeverBeUsedInProduction
 
 service:
-  annotations:
-    service.kubernetes.io/topology-aware-hints: "auto"
+  trafficDistribution: PreferClose
```

```suggestion
  trafficDistribution: PreferSameZone
```

https://kubernetes.io/docs/reference/networking/virtual-ips/#traffic-distribution

> This means the same thing as PreferClose, but is more explicit. (Originally, the intention was that PreferClose might later include functionality other than just "prefer same zone", but this is no longer planned. In the future, PreferSameZone will be the recommended value to use for this functionality, and PreferClose will be considered a deprecated alias for it.)